### DISCUSSION TITLE
In Timesheets, pin the button bar at the top of the page.

### DISCUSSION THREAD

**Discussion by WhiteTiger-IT** [kimai/kimai#2453] [General]
Sometimes you are working on the last lines of the Timesheets page and if you need to create a new Job, you have to run the page backwards before the buttons at the top right appear again.
I suggest fixing that bar at the top or inserting a button at the bottom that allows you to quickly return to the top, as for example you can find in many blogs.

**Comment by kevinpapst** [comment]
I really like the idea and already worked on it. 

But I am stuck with a problem in regard to the different layout options:
there is the boxed and fullscreen layout, collapsed and expanded navigation and all the responsive (mobile) view as well.

All of them need special treatment and there is one issue that I can't get rid of (the collapsing of the navigation bar and jumping buttons). 

I will push a PR soon and see if someone else has an idea.
